### PR TITLE
Add/required questions face form

### PR DESCRIPTION
**Pull request by anabebs**
**What**  
Make Face overview Form prod ready
**Why**  
Added missing checkbox questions
Added missing paragraphs and added an intro field to make this possible. 
added required questions
rephrased some questions
added conditionals
collapsed whole sections in order for conditionals to work.

### CODE REVIEW

File: pages/submissions/[id]/index.tsx [hunk 1]
```diff
@@ -104,15 +104,6 @@ export const getServerSideProps: GetServerSideProps = async ({ params }) => {
       },
     };
 
-  if (data.submissionState !== 'In progress') {
```
Comment (jhackett1) [inline pages/submissions/[id]/index.tsx:107]: where did this go?

Comment (anabebs) [reply pages/submissions/[id]/index.tsx:107]: I didn't do that.

Comment (jhackett1) [reply pages/submissions/[id]/index.tsx:107]: let's bring it back then - can you make this file the way it was pre-changes?

**Review by jhackett1 — APPROVED** [review]
just gotta fix the redirect code that got undone

File: pages/submissions/[id]/index.tsx (hunk 1 above)
Comment (anabebs) [reply pages/submissions/[id]/index.tsx]: ok

**Pull request merged**